### PR TITLE
docs: update running-browser-tests.md

### PR DESCRIPTION
**Pull request by MoKhajavi75**
## What?

Use more general term for new Macbooks.

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.
- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [x] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

## Related PR(s)/Issue(s)

None

### CODE REVIEW

File: docs/sources/next/using-k6-browser/running-browser-tests.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ To run a simple local script:
 
    {{% /admonition %}}
 
-1. Optional step: running browser tests in Docker on M1/M2 Macs
+1. Optional step: running browser tests in Docker on Macbooks with Apple Silicon:
```
Comment (heitortsergent) [inline docs/sources/next/using-k6-browser/running-browser-tests.md:147]:
```suggestion
1. Optional step: running browser tests in Docker on Mac computers with Apple Silicon:
```

File: docs/sources/v0.49.x/using-k6-browser/running-browser-tests.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ To run a simple local script:
 
    {{% /admonition %}}
 
-1. Optional step: running browser tests in Docker on M1/M2 Macs
+1. Optional step: running browser tests in Docker on Macbooks with Apple Silicon:
```
Comment (heitortsergent) [inline docs/sources/v0.49.x/using-k6-browser/running-browser-tests.md:147]:
```suggestion
1. Optional step: running browser tests in Docker on Mac computers with Apple Silicon:
```

File: docs/sources/v0.50.x/using-k6-browser/running-browser-tests.md [hunk 3]
```diff
@@ -144,7 +144,7 @@ To run a simple local script:
 
    {{% /admonition %}}
 
-1. Optional step: running browser tests in Docker on M1/M2 Macs
+1. Optional step: running browser tests in Docker on Macbooks with Apple Silicon:
```
Comment (heitortsergent) [inline docs/sources/v0.50.x/using-k6-browser/running-browser-tests.md:147]:
```suggestion
1. Optional step: running browser tests in Docker on Mac computers with Apple Silicon:
```

File: docs/sources/v0.51.x/using-k6-browser/running-browser-tests.md [hunk 4]
```diff
@@ -144,7 +144,7 @@ To run a simple local script:
 
    {{% /admonition %}}
 
-1. Optional step: running browser tests in Docker on M1/M2 Macs
+1. Optional step: running browser tests in Docker on Macbooks with Apple Silicon:
```
Comment (heitortsergent) [inline docs/sources/v0.51.x/using-k6-browser/running-browser-tests.md:147]:
```suggestion
1. Optional step: running browser tests in Docker on Mac computers with Apple Silicon:
```

File: docs/sources/v0.52.x/using-k6-browser/running-browser-tests.md [hunk 5]
```diff
@@ -144,7 +144,7 @@ To run a simple local script:
 
    {{% /admonition %}}
 
-1. Optional step: running browser tests in Docker on M1/M2 Macs
+1. Optional step: running browser tests in Docker on Macbooks with Apple Silicon:
```
Comment (heitortsergent) [inline docs/sources/v0.52.x/using-k6-browser/running-browser-tests.md:147]:
```suggestion
1. Optional step: running browser tests in Docker on Mac computers with Apple Silicon:
```

**Review by heitortsergent — CHANGES_REQUESTED** [review]
@MoKhajavi75 thanks for your contribution! I made a small change since we can have other Mac computers with Apple Silicon besides MacBooks.

**Comment by MoKhajavi75** [comment]
Totally missed that point 😁 
Thanks & Done @heitortsergent

**Review by heitortsergent — APPROVED** [review]
LGTM! 🚀  Thanks for your contribution! 🙇

**Pull request merged**